runtime/hydradx/src/lib.rs

```diff
@@ -382,7 +381,7 @@ impl pallet_session::Config for Runtime {
 	type ValidatorIdOf = pallet_collator_selection::IdentityCollator;
 	type ShouldEndSession = pallet_session::PeriodicSessions<Period, Offset>;
 	type NextSessionRotation = pallet_session::PeriodicSessions<Period, Offset>;
-	type SessionManager = CollatorSelection;
+	type SessionManager = CollatorRewards;
```

Comment:
Implemented `SessionManager` on the pallet directly. Could move it to a dedicated type instead. (As this is the central piece of functionality of the pallet I don't think it's necessary, though.)

Reply:
yeah, this is okay